qplaintexteditsearchwidget.cpp

```diff
@@ -26,16 +26,16 @@ QPlainTextEditSearchWidget::QPlainTextEditSearchWidget(QPlainTextEdit *parent) :
     _textEdit = parent;
     _darkMode = false;
     hide();
-    ui->searchCountLabel->setStyleSheet("* {color: grey}");
+    ui->searchCountLabel->setStyleSheet(QStringLiteral("* {color: grey}"));
     // hiding will leave a open space in the horizontal layout
     ui->searchCountLabel->setEnabled(false);
     _currentSearchResult = 0;
     _searchResultCount = 0;
 
     QObject::connect(ui->closeButton, SIGNAL(clicked()),
                      this, SLOT(deactivate()));
-    QObject::connect(ui->searchLineEdit, SIGNAL(textChanged(const QString &)),
-                     this, SLOT(searchLineEditTextChanged(const QString &)));
+    QObject::connect(ui->searchLineEdit, SIGNAL(textChanged(QString)),
+                     this, SLOT(searchLineEditTextChanged(QString)));
```

Comment:
Does this still work the same way? :grin:

Reply:
Haha yes, that's signature normalization. A performance penalty is paid if you don't do it because then the Qt Moc has to do it.

Reply:
Here's a link to more explanation if you are interested:
https://marcmutz.wordpress.com/effective-qt/prefer-to-use-normalised-signalslot-signatures/

It has some other gems too ;)